src/main/java/org/folio/circulation/resources/RequestNoticeSender.java

```diff
@@ -47,24 +47,25 @@ public RequestNoticeSender(PatronNoticeService patronNoticeService, RequestRepos
     this.requestRepository = requestRepository;
   }
 
-
-  public Result<RequestAndRelatedRecords> sendNoticeOnRequestCreated(
+  public Result<RequestAndRelatedRecords> sendNoticeOnRequestCreatedOrMoved(
     RequestAndRelatedRecords relatedRecords) {
 
     Request request = relatedRecords.getRequest();
-    Item item = request.getItem();
-    User requester = request.getRequester();
-    NoticeEventType eventType =
-      requestTypeToEventMap.getOrDefault(request.getRequestType(), NoticeEventType.UNKNOWN);
 
-    PatronNoticeEvent requestCreatedEvent = new PatronNoticeEventBuilder()
-      .withItem(item)
-      .withUser(requester)
-      .withEventType(eventType)
-      .withTiming(NoticeTiming.UPON_AT)
-      .withNoticeContext(createRequestNoticeContext(request))
-      .build();
-    patronNoticeService.acceptNoticeEvent(requestCreatedEvent);
+    if (!relatedRecords.isMoveRequest()) {
```

Comment:
If we are introducing branching for two different circumstances, `create` and `move`, could we instead introduce a new method for `move`?

Reply:
No problem. I will create a move method and extract common code into another private method.